app/src/main/java/com/vultisig/wallet/ui/models/referral/EditVaultReferralViewModel.kt

```
@@ -0,0 +1,269 @@
+                    ),
+                    estimatedFees = gasFees,
+                    estimateFeesFiat = gasFees.value.convertToFiat(),
+                    blockChainSpecific = blockchainSpecific,
+                )
+
+                transactionRepository.addTransaction(tx)
+
+                navigator.route(
+                    Route.VerifyDeposit(vaultId, tx.id)
+                )
+            } catch (t: Throwable) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)